newIDE/app/src/SceneEditor/index.js

```diff
@@ -223,6 +228,10 @@ export default class SceneEditor extends React.Component<Props, State> {
       <Toolbar
         showPreviewButton={this.props.showPreviewButton}
         onPreview={() => this.props.onPreview({})}
+        isPreviewOverride={this.props.isPreviewOverride}
+        togglePreviewOverride={this.props.togglePreviewOverride}
+        previewFirstSceneName={this.props.previewFirstSceneName}
+        setScenePreview={() => this.props.setScenePreview({})}
```

Comment:
I don't think you need to pass {} as argument? There is no argument at all needed?